lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -193,7 +193,7 @@ sub info ($self) {
 
 sub send_command {
     my ($self, %args) = @_;
-    return undef if (!defined $args{command});
+    return undef unless defined $args{command} && defined $self->host && defined $self->instance;
```

Comment:
We should probably prevent the registration of workers with empty `host` and `instance` names in the first place. (And with empty I mean empty. Your check is about whether those values are defined at all.)

Note that in theory `$self->host` and `$self->instance` are already supposed to be never undefined because those columns are not nullable. We should rather find out why `$self->instance` was rendered as empty string here and fix a potential bug elsewhere.

I also doubt that the websocket send attempt was invalid here. We don't need an instance number for this to work. The `Connection refused` error indicates that the web socket server was not responding which has nothing to do with the concrete worker the command is sent to.